### ISSUE TITLE
Unable to clear value of custom field

### ISSUE THREAD

**Issue by waza-ari** [waza-ari/python-easyverein#4]
When trying to delete the value of a custom field (e.g. setting the value to `""`), the API throws a 500 error.

Relevant logs:

```
Jan 21 02:04:29 keybox main.py[1908]: Updating object of type member/<member_id>/custom-fields with id <cf_id>
Jan 21 02:04:29 keybox main.py[1908]: Base URL is https://hexa.easyverein.com/api/v1.7/member/<member_id>/custom-fields/<cf_id>
Jan 21 02:04:29 keybox main.py[1908]: Final constructed URL is https://hexa.easyverein.com/api/v1.7/member/<member_id>/custom-fields/<cf_id>
Jan 21 02:04:29 keybox main.py[1908]: Performing patch request to https://hexa.easyverein.com/api/v1.7/member/< member_id>/custom-fields/<cf_id>
Jan 21 02:04:29 keybox main.py[1908]: Final request headers: {'Authorization': 'Bearer <token>'}
Jan 21 02:04:30 keybox main.py[1908]: Request returned status code 500
```

Note the absence of `Request data: {'value': 'xyz'}` log, indicating the library sends a PATCH request with empty data. This might be caused by Pydantics behaviour during model dump, where an empty string might be removed when `exclude_unset = True`.

**Comment by waza-ari** [comment]
This is actually caused by the `EmptyStringsToNone` class [which all models inherit from](https://github.com/waza-ari/python-easyverein/blob/08836834631b6693fc7e1ecd8cb510365d6f7a6e/easyverein/models/base.py#L7). In a second step, the attribute gets filtered out by `model_dump` attributes: https://github.com/waza-ari/python-easyverein/blob/08836834631b6693fc7e1ecd8cb510365d6f7a6e/easyverein/core/client.py#L188-L190

This is meant to simplify working with the library but might not be desired in all cases - at the very least it needs to be refactored to only take effect when reading data, not when writing or updating data.

**Comment by waza-ari** [comment]
Fixed in [release v0.2.4](https://github.com/waza-ari/python-easyverein/releases/tag/v0.2.4).
